src/components/app/service.ts

```diff
@@ -45,6 +45,11 @@ export function getCITriggerInfoModal(
 ) {
     return getCITriggerInfo(params).then((response) => {
         let materials = response?.result?.ciMaterials || []
+        let appReleaseTags = response?.result?.imageTaggingData?.appReleaseTags
```

Comment:
we should use these values directly, no need to create variables

Reply:
added these for readability